website/content/docs/upgrading/version-guides/0.11.0.mdx

```diff
@@ -0,0 +1,92 @@
+---
+layout: docs
+page_title: Version 0.11.0 - Upgrade Guide
+description: |-
+This upgrade guide describes any considerations for upgrading to Waypoint 0.11.0.
+---
+
+# Upgrading to Waypoint 0.11.0 from 0.10.x
+
+Waypoint 0.10.x can be upgraded to Waypoint 0.11.0 using a [standard upgrade](/waypoint/docs/upgrading#standard-upgrade).
+There are some considerations depending on what platform your server is installed on.
+There are no breaking changes in the protocol or `waypoint.hcl` configuration file.
+Due to the changes in `runner` management, we cannot guarantee that all Waypoint
+components (CLI, entrypoint, and server) are forwards and backwards
+compatible. We encourage users to have all components on the same version.
+
+## Server Snapshot
+
+Waypoint servers at versions 0.10.4 and 0.10.5 have a bug with taking snapshots,
+if there is a dynamic configuration source set, using the `config source-set`
+command. If your server is running at either of these versions and a config
+source is set, then you will not be able to take snapshots using the command
+`server snapshot`. By default, the `server upgrade` command will also take a
+snapshot of the database, and this also fails due to this bug.
+
+There are a few options to upgrade to v0.11.0. If you do not need a snapshot of
+the database before upgrading, then you may run `server upgrade` with the flag
+`-snapshot=false` to proceed without issue.
+
+Another alternative is to delete all config sources in your database using
+`config source-set` with the `-delete` flag, and then running `server upgrade`.
+After the server is upgraded, you can re-create the config sources using
+`config source-set`.
```

Comment:
I'd recommend showing the full command in a code block with an example source-set here so it's easy at a glance to see what people will have to do to fix these configs.